yb-voyager/cmd/exportData.go

```diff
@@ -74,10 +71,10 @@ func exportData() {
 	utils.PrintAndLog("export of data for source type as '%s'", source.DBType)
 
 	var success bool
-	if migrationMode == "offline" {
-		success = exportDataOffline()
-	} else {
+	if migrationMode == "online" {
 		success = exportDataOnline()
+	} else {
+		success = exportDataOffline()
```

Comment:
Get rid of the `exportDataOnline()` function and the `migrationMode` variable.

Reply:
Done